verification/certifyKey_test.go

```diff
@@ -118,21 +579,148 @@ func testCertifyKey(d TestDPEInstance, t *testing.T) {
 	}
 	client, err := NewClient256(d)
 	if err != nil {
-		t.Fatalf("Could not initialize client: %v", err)
+		t.Fatalf("[FATAL]: Could not initialize client: %v", err)
+	}
+
+	var ctx ContextHandle
+	var initCtxResp *InitCtxResp
+
+	if d.GetSupport().Simulation {
```

Comment:
This still just checks if the DPE instance supports simulation contexts, not whether the test is expecting to use simulation contexts. I would expect `testCertifyKey` to accept a boolean `use_simulation`